File: app/src/main/java/com/jameskbride/eventbusmvpdemo/utils/ToasterWrapper.java
package com.jameskbride.eventbusmvpdemo.utils;

import android.content.Context;
import android.support.annotation.StringRes;
import android.widget.Toast;

public class ToasterWrapper {

    private Toast toast;

    public ToasterWrapper makeText(Context context, @StringRes int message, int duration) {
        this.toast = Toast.makeText(context, message, duration);

        return this;
    }

    public void show() {
        toast.show();
    }
}

File: app/src/main/java/com/jameskbride/eventbusmvpdemo/utils/ToasterWrapper.kt
package com.jameskbride.eventbusmvpdemo.utils

import android.content.Context
import android.widget.Toast
import androidx.annotation.StringRes

class ToasterWrapper {

    private lateinit var toast: Toast

    fun makeText(context: Context, @StringRes message: Int, duration: Int):ToasterWrapper {
        this.toast = Toast.makeText(context, message, duration)

        return this
    }

    fun show() {
        toast.show()
    }
}